src/main/java/com/redhat/cloud/common/clowder/configsource/ClowderConfigSource.java

```diff
@@ -251,4 +263,18 @@ private String getHostPortDb(JsonObject dbObject) {
                 port,
                 dbName);
     }
+
+    private String createTempCertFile(JsonObject dbObject) {
+        if (dbObject.containsKey("rdsCa")) {
+            byte[] cert = dbObject.getString("rdsCa").getBytes(UTF_8);
+            try {
+                File certFile = File.createTempFile("rds-ca-root", ".crt");
```

Comment:
Should we
* use `Files.createTempFile`
* add a `deleteOnExit()` call -- even if the container should be garbage collected anyway
?

Reply:
Good idea, I added the `deleteOnExit` call.